screens/Matches.js

```diff
@@ -1,23 +1,93 @@
 import React from 'react';
 import styles from '../styles'
+import { connect } from 'react-redux';
+import * as firebase from 'firebase';
 
 import { 
-  Text, 
+  Text,
+  ScrollView,
+  TouchableOpacity,
+  Image,
   View
 } from 'react-native';
 
-class Matches extends React.Component {
-  state = {}
+function intersect(a, b) {
+    var t;
+    if (b.length > a.length) t = b, b = a, a = t; // indexOf to loop over shorter
+    return a.filter(function (e) {
+        return b.indexOf(e) > -1;
+    });
+  }
+
+function retrieveswipesyes (userid) {
```

Comment:
J'ai du mal à comprendre certaines de tes fonctions

Le must pour que je t'aide : donne moi un exemple de données d'entrée, et des données de sortie (comme si tu écrivais un test unitaire 😉 ) pour toutes les fonctions de traitement de données

Reply:
C'est la partie ou je suis un peu bloqué. 
Au final ce que je voudrais c'est une fonction qui me donne les cartes à l'intersection entre les swipesyes des deux utilisateurs de la room.
J'ai décomposé cela en plusieurs étapes:
Step 1: retrieve Swipesyes de l'utilisateur logged (c'est la fonction que tu as commenté ici que je n'ai pas encore trouvé comment faire"
Step 2: retrieve userid de l'autre personne dans la room
Step 3: retrieve swipesyes de l'user2
Step 4: get intersection
Step 5: retrieve text for cardsid de l'intersection
Step 6: display

Reply:
Ok ! Je vais me pencher sur cette question demain 😊
Le must serait peut-être un appel avec partage d'écran !